doc/web-doc/README.md

```diff
@@ -0,0 +1,47 @@
+# Documentation framework for Theia Trace Extension
+
+## Installation
+
+Requirements: NodeJS on the same version as the one required for [Theia IDE][theia-prereq].
+
+```bash
+cd doc/web-doc
+yarn
+```
+
+> Executing `yarn` from the root of theia-trace-extension does not install the dependencies in doc/web-doc. The configuration for Lerna to support such feature may be added in the future.
```

Comment:
1. `theia-trace-extension` (marked as `code`), too?
2. I'd also format `doc/web-doc` the same way.
3. I'd write `such *a* feature`.

Reply:
Done.